docs/melange-book/index.md

```diff
@@ -0,0 +1,16 @@
+# Melange Book
```

Comment:
Should we add your table with the Chapter and Topics columns somewhere here? I think it might be useful for readers to see what topics they can learn on each chapter? 🤔